configure.ac

```diff
@@ -1201,6 +1210,32 @@ AS_IF([test "x$build_libuuid" = xyes], [
   AC_DEFINE(HAVE_LIBUUID, 1, [Define to 1 if you have the -luuid.])
 ])
 
+dnl
+dnl liblastlog2
```

Comment:
Also add sqlite3 to `.packit.yaml` to test it there.

Reply:
I have adde sqlite-devel